conf.py

```diff
@@ -56,8 +56,8 @@
 
 extlinks = {
     'issue': ('https://jira.mongodb.org/browse/%s', '' ),
-    'manual': ('http://docs.mongodb.org/manual%s', ''),
-    'php': ('http://php.net/%s', '')
+    'manual': ('https://www.mongodb.com/docs/manual%s', ''),
```

Comment:
Could you add the `docs` prefix for the need of https://github.com/mongodb/mongo-php-library/pull/1273#discussion_r1559471310

```suggestion
    'manual': ('https://www.mongodb.com/docs/manual%s', ''),
    'docs': ('https://www.mongodb.com/docs%s', ''),
```

Reply:
Would `:atlas:` make more sense? I see that is used in some existing docs outside of PHPLIB. For instance, the "Connect from Your Application" link in [Python Drivers: Compatibility](https://www.mongodb.com/docs/drivers/python-drivers/#compatibility) uses that (see: [here](https://github.com/mongodb/docs-ecosystem/blob/2a75928245e315b64f86f252d8cfcf6ad039e7f2/source/python-drivers.txt#L51)).

There are definitely other, existing links to https://www.mongodb.com/docs/atlas/ throughout the PHPLIB docs that could be changed to take advantage of that.

Will defer to @ccho-mongodb here.

Reply:
Yes! I didn't think to check how it was configured elsewhere.
```suggestion
    'manual': ('https://www.mongodb.com/docs/manual%s', ''),
    'atlas': ('https://www.mongodb.com/docs/atlas%s', ''),
```